ski_conditions/apps/app_scraping/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+        path('', views.index, name='index'),
+        ]
```

Comment:
Correct the spacing. This closing bracket should not have any whitespace in front of it.

Reply:
Done